gemet/thesaurus/import_spreadsheet.py

```diff
@@ -0,0 +1,304 @@
+from openpyxl import load_workbook
+from openpyxl.utils.exceptions import InvalidFileException
+
+from django.db import transaction
+from django.utils import timezone
+
+from gemet.thesaurus import PENDING, PUBLISHED
+from gemet.thesaurus.models import (
+    Namespace, Version, Concept, Property, PropertyType, Relation, Language
+)
+from gemet.thesaurus.utils import get_new_code, get_search_text
+
+
+class ImportError(Exception):
+    pass
+
+
+def namespace_for(property_type_name):
+    heading_for = {
+        'broader': 'Concepts',
+        'group': 'Groups',
+    }
+    return Namespace.objects.get(
+        heading=heading_for[property_type_name]
+    )
+
+
+def row_dicts(sheet):
+    rows = sheet.rows
+
+    mandatory_columns = {
+        "Term", "Definition", "Definition reference"
+    }
+    optional_columns = {
+        "Alt Label", "Abbreviation/Alt Label", "Synonym/Alt Label",
+        "Broader concept", "Broader URI", "Group", "Note"
```

Comment:
`Broader URI` is not actually used anywhere during the import. Better remove it from the template, to avoid confusion.  "Broader prefLabel" should be enough